docs/user-guide/ze-usage.md

```diff
@@ -6,23 +6,24 @@ Review this section to familiarize yourself with Zowe Explorer and make the best
 
 ### Operating systems
 
-- Windows 10+
+
 - MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey)
 - Unix-like:
    - [CentOS](https://www.centos.org/) 8+
    - [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux) 8+
    - [Ubuntu](https://ubuntu.com/) 20.04+
```

Comment:
We may want to also add that we support Debian 11 (bullseye). And for all of these operating systems, we need to disclaim that we only support x64, and features may not function correctly on other CPU architectures (i.e. x86, arm64, MIPS, s390(x)...)

As far as I am aware, the CLI works in most of those environments, but Daemon mode does not.

Reply:
To complicate matters, we also need to disclaim that we do support running on MacOS with the M1/M2/M3 processor with Rosetta, UNLESS they want the DB2 plug-in, in which case the Intel version of Node is required.

Reply:
Well, this thread was meant to be about the CLI, and I appear to have accidentally put it on the Zowe Explorer and cannot move the conversation. Oops. :/

I think, if we have a section, we are just going to say that the CLI supports x86_64 based processors on Windows 10, Ubuntu 20.04+, Debian 11+, RHEL 8+, CentOS 8+, and MacOS 10.15+.

Reply:
Feedback incorporated into the cli-using-usingcli.md file -- thank you!